src/plugins/workspace/public/components/workspace_creator/workspace_form.tsx

```diff
@@ -129,8 +134,9 @@ export const WorkspaceForm = ({
   const [color, setColor] = useState(defaultValues?.color);
   const [icon, setIcon] = useState(defaultValues?.icon);
   const [defaultVISTheme, setDefaultVISTheme] = useState(defaultValues?.defaultVISTheme);
-
   const isEditingManagementWorkspace = defaultValues?.id === MANAGEMENT_WORKSPACE_ID;
+  const [tabFeatureSelected, setTabFeatureSelected] = useState(!isEditingManagementWorkspace);
```

Comment:
Prefer change this state to a string, and record selected tab here.

Reply:
There is going to be a third tab called **workspace settings** in update PR. Can I include this change in my next PR?